### PR TITLE
feat(Team&Project): add team and project list

### PR DESCRIPTION
**Pull request by gavin1ee**
add team and project list

### CODE REVIEW

File: src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Masa.Scheduler.ApiGateways.Caller.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Masa.Utils.Caller.HttpClient" Version="0.3.0" />
+    <PackageReference Include="Masa.Utils.Caller.HttpClient" Version="0.4.0-preview.4" />
```
Comment (MayueCif) [inline src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Masa.Scheduler.ApiGateways.Caller.csproj:10]: update rc

File: src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/SchedulerServerCaller.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    AuthService? _authService;
+    PMService? _pmService;
+
+    public SchedulerJobService JobService => _jobService ?? (_jobService = new(CallerProvider));
+
+    public AuthService AuthService => _authService ?? (_authService = new(CallerProvider));
+
+    public PMService PMService => _pmService ?? (_pmService = new(CallerProvider));
+
+    public SchedulerServerCaller(IServiceProvider serviceProvider, SchedulerApiOptions options) : base(serviceProvider)
```
Comment (MayueCif) [inline src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/SchedulerServerCaller.cs:18]: IOptions<SchedulerApiOptions>

File: src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/ServiceCollectionExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+namespace Masa.Scheduler.ApiGateways.Caller;
+
+public static class ServiceCollectionExtensions
+{
+    public static IServiceCollection AddSchedulerApiGateways(this IServiceCollection services, Action<SchedulerApiOptions>? configs = null)
+    {
+        var option = new SchedulerApiOptions("http://localhost:16002");
```
Comment (MayueCif) [inline src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/ServiceCollectionExtensions.cs:9]: should remove hard code,masa.auth is wrong demo

File: src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Services/AuthService.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+public class AuthService : ServiceBase
+{
+    protected override string BaseUrl { get; set; }
+
+    public AuthService(ICallerProvider provider) : base(provider)
+    {
+        BaseUrl = "api/auth";
+    }
+    public async Task<List<TeamModel>> GetTeamListAsync()
```
Comment (MayueCif) [inline src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Services/AuthService.cs:14]: Add Masa.BuildingBlocks.Service.Contracts ,user XXXXRequest and XXXResponse

File: src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Services/PMService.cs [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+public class PMService: ServiceBase
+{
+    protected override string BaseUrl { get; set; }
+
+    public PMService(ICallerProvider provider) : base(provider)
+    {
+        BaseUrl = "api/pm";
+    }
+    public async Task<List<ProjectModel>> GetProjectListAsync(Guid teamId)
```
Comment (MayueCif) [inline src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Services/PMService.cs:14]: ProjectModel modify

File: src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Services/SchedulerJobService.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+public class SchedulerJobService : ServiceBase
+{
+    protected override string BaseUrl { get; set; }
+
+    public SchedulerJobService(ICallerProvider callerProvider) : base(callerProvider)
+    {
+        BaseUrl = "api/job/";
+    }
+
+    public async Task<List<SchedulerJobDto>> GetListAsync()
```
Comment (MayueCif) [inline src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Services/SchedulerJobService.cs:15]: Add SchedulerJobResponse

File: src/Services/Masa.Scheduler.Services.Server/Application/Teams/TeamQueryHandler.cs [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+            {
+                Id= new Guid("3119137C-DB47-4523-C509-08DA1EC03F6F"),
+                Name = "SEC团队",
+                Description ="SEC 团队",
+                MemberCount = 3,
+            }
+        };
+
+        return Task.CompletedTask;
+
```
Comment (MayueCif) [inline src/Services/Masa.Scheduler.Services.Server/Application/Teams/TeamQueryHandler.cs:37]: delete blank line

File: src/Web/MASA.Scheduler.Web.Admin/Pages/Teams/Team.razor [hunk 8]
```diff
@@ -0,0 +1,33 @@
+﻿@inherits ProCompontentBase
+@page "/team/{teamId}"
+
+<MTabs @bind-Value="_curTab" SliderColor="transparent" class="pro-nav remover-ripple">
+
```
Comment (MayueCif) [inline src/Web/MASA.Scheduler.Web.Admin/Pages/Teams/Team.razor:5]: delete blank line

File: src/Web/MASA.Scheduler.Web.Admin/Pages/Teams/Team.razor.cs [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+public partial class Team
+{
+
+    [Parameter]
+    public string TeamId { get; set; } = string.Empty;
+
+    private int _projectId;
+    private StringNumber _curTab = 0;
+
```
Comment (MayueCif) [inline src/Web/MASA.Scheduler.Web.Admin/Pages/Teams/Team.razor.cs:14]: delete line

**Review by MayueCif — CHANGES_REQUESTED** [review]
modify with conversation

**Review by doddgu — APPROVED** [review]
.

**Review by MayueCif — APPROVED** [review]

**Comment by MayueCif** [comment]
@doddgu  LGTM

File: src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Services/SchedulerJobService.cs [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+namespace Masa.Scheduler.ApiGateways.Caller.Services;
+
+public class SchedulerJobService : ServiceBase
+{
+    protected override string BaseUrl { get; set; }
+
+    public SchedulerJobService(ICallerProvider callerProvider) : base(callerProvider)
+    {
+        BaseUrl = "api/job/";
```
Comment (doddgu) [inline src/ApiGateways/Caller/Masa.Scheduler.ApiGateways.Caller/Services/SchedulerJobService.cs:12]: scheduler job? job?

File: src/Services/Masa.Scheduler.Services.Server/Services/SchedulerJobService.cs [hunk 11]
```diff
@@ -5,22 +5,20 @@ namespace Masa.Scheduler.Services.Server.Server.Services;
 
 public class SchedulerJobService : ServiceBase
 {
-    public SchedulerJobService(IServiceCollection services) : base(services)
+    public SchedulerJobService(IServiceCollection services) : base(services, "api/job")
```
Comment (doddgu) [inline src/Services/Masa.Scheduler.Services.Server/Services/SchedulerJobService.cs:8]: scheduler job? job?

**Review by doddgu — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**